src/grokcore/component/tests/test_grok.py

```diff
@@ -38,7 +43,8 @@ def suiteFromPackage(name):
                                         tearDown=cleanUpZope,
                                         checker=checker,
                                         optionflags=doctest.ELLIPSIS+
-                                        doctest.NORMALIZE_WHITESPACE)
+                                        doctest.NORMALIZE_WHITESPACE+
+                                        doctest.IGNORE_EXCEPTION_DETAIL)
```

Comment:
Ugh.  I know this was in the original, but using `+` instead of `|` for bit flags feels wrong.